Lib/encodings/aliases.py

```diff
@@ -439,6 +443,7 @@
 
     # mac_cyrillic codec
     'maccyrillic'        : 'mac_cyrillic',
+    'x_mac_cyrillic'     : 'mac_cyrillic',
```

Comment:
Why x_mac_cyrillic, but not x_mac_greek etc?

Reply:
Because the issue was only about cyrillic. I think that we should implement all the aliases but I'm not sure if I should do it in a PR related to a issue that cited only those two. I can implement the others if you think that's ok to do it in this PR